.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           path: /tmp/heroku-${{ matrix.stack-version }}*
       - name: Upload heroku runtime images to staging
         run: |
-          bin/upload-runtime-images.sh ${{ matrix.stack-version }} ${{ github.sha }}
+          bin/upload-runtime-images.sh ${{ matrix.stack-version }} ${{ github.ref_name }}
```

Comment:
The `ctc-check` job also uses `github.sha` (when iirc that used the tag before too). Should that be switched back to the tag too?

Reply:
@joshwlewis Just checking you saw ^ ? :-)